### ISSUE TITLE
Cannot read property 'windowHeight' of null

### ISSUE THREAD

**Issue by Sapphire64** [jossmac/react-images#32]
Hello again. Currently I'm updating my project to React 14 and had to update this library as well, but I'm getting this:

![screen shot 2016-03-11 at 13 59 43](https://cloud.githubusercontent.com/assets/1443567/13702936/60de44ac-e792-11e5-865c-fdafb51f4456.png)

#### Possible reasons

It looks like we set `windowHeight` in state but `setState` is async, so according to the docs it's possible that we will have it undefined when we start to render. At least I believe this is my case. My code looks like this:
```
    return (
      <Lightbox
        images={images.map((image) => {return {src: image}})}
        isOpen={true}
        onClose={this.onClose.bind(this)}
        onClickPrev={() => {}}
        onClickNext={() => {}}
      />
    );
```
As a fix I've downgraded to version `0.2.0` and now it works perfect with my older setup, which is this:
```
return (
      <Lightbox
        images={images.map((image) => {return {src: image}})}
        isOpen={true}
        onClose={this.onClose.bind(this)}
        onClickPrev={() => {}}
        onClickNext={() => {}}
        styles={this.props.styles}
        width={width}
        height={height}
      />
    );
```

Yes, I want to render only one image :)

#### Questions

1. Why don't we read windowHeight from window on each render, what's the reason to keep it in state? If you think it makes sense to change the way we do now I may have time to take a look on weekends :)
2. Why `onClickPrev` and `onClickNext` are required even if you have only one image?

**Comment by yangligogogo** [comment]
I have this problem too. Version is 0.3.2,  React version is 0.14.6

**Comment by jossmac** [comment]
Thanks for the details guys. I think I'll re-write the layout styles for this; not really comfortable requiring internal state for such a simple component.

@Sapphire64 good point, i'll remove the requirement for next/prev functions and handle the exceptions. Did you have a look into removing state?

**Comment by jossmac** [comment]
Fixed by #38
